applications/openshift/logging/cluster_logging_operator_exist/rule.yml

```diff
@@ -0,0 +1,47 @@
+prodtype: ocp4
+
+title: Ensure that OpenShift Logging Operator is scanning the cluster
+
+description: |-
+  {{{ weblink(link="https://docs.openshift.com/container-platform/4.10/logging/cluster-logging.html",
+              text="The OpenShift Logging Operator") }}}
+  OpenShift Logging Operator provides ability to aggregate all the logs from the
+  OpenShift Container Platform cluster, such as node system audit logs, application
+  container logs, and infrastructure logs. OpenShift Logging aggregates these logs
+  from throughout OpenShift cluster and stores them in a default log store. [1]
+
+rationale: |-
+  OpenShift Logging Operator is able to collect, aggregate, and manage logs.
```

Comment:
We already have a rule `audit_log_forwarding_enabled`. I'm not against this rule in principle, but is it even possible to have the log forwarding enabled but having this rule fail?

Reply:
I think it might be possible?